tools/scripts/AksEdgeQuickStart/AksEdgeQuickStartForAio.ps1

```diff
@@ -12,11 +12,89 @@ param(
     [String] $ResourceGroupName,
     [ValidateNotNullOrEmpty()]
     [String] $ClusterName,
+    [String] $CustomLocationOid,
     [Switch] $UseK8s=$false,
     [string] $Tag
 )
 #Requires -RunAsAdministrator
-New-Variable -Name gAksEdgeQuickStartForAioVersion -Value "1.0.240419.1300" -Option Constant -ErrorAction SilentlyContinue
+
+function Verify-ConnectedStatus
+{
+param(
+    [Parameter(Mandatory=$true)]
+    [object] $arcArgs,
+    [Parameter(Mandatory=$true)]
+    [string] $clusterName
+)
+
+    $k8sConnectArgs = @("-g", $arcArgs.ResourceGroupName)
+    $k8sConnectArgs += @("-n", $clusterName)
+    $k8sConnectArgs += @("--subscription", $arcArgs.SubscriptionId)
+
+    $retries = 150
```

Comment:
can we define the max retry and interval as a global constant and leave some comments on why we should wait for 25m?

Reply:
15m should be sufficient, will add appropriate comments there - thats the recommended timeout from Arc team.

Reply:
prefer to keep the retries local - it is not used elsewhere...